### ISSUE TITLE
ZFS

### ISSUE THREAD

**Issue by lawmanuk** [Le0xFF/VoidLinuxInstaller#20]
Hi,

Could you please give the user an option to install on ZFS instead of BTRFS.    

Would love to play with ZFS as an advanced system.

thanks

**Comment by Le0xFF** [comment]
I really want to avoid using ZFS to avoid problem related to kernel upgrades.

I never used ZFS, so I have complitely no knownledge about it.
To implement such feature I should get deeper into the ZFS filesystem, undestand how it deals with subvolumes and all the other funcionalities.
Then, part of the code must be extended and at the moment i see that as a big effort that I have no time to spend on.

Long story short: the feature is not planned, but I will keep the issue as open, just in case.
